### DISCUSSION TITLE
Cant get SCUMMVM games to show up in simple menu

### DISCUSSION THREAD

**Discussion by lagwagone** [eduardofilo/RG350_adam_image#91] [General]
Would it be possible to update the readme to detail how this is supposed to work? I've tried adding a blank .svn file in the game folder and adding the name to alias.txt but had no luck. 

Eg. for The Curse of Monkey Island - taken game short name from https://www.scummvm.org/compatibility/

Added "comi.svn" (blank file) to roms/SCUMMVM/The Curse Of Monkey Island/ 
Added "comi=The Curse Of Monkey Island" to alias.txt
The SCUMMVM section doesn't show up in the Home Computers section

**Comment by aplanas** [comment]
Did you follow this? https://www.reddit.com/r/RG350/comments/pj9i6y/how_to_setup_scummvm_in_adam/

**Comment by lagwagone** [comment]
@aplanas followed that guide but using WinSCP, but I still don't see a SCUMMVM section in home computers. I only see ZX spectrum under home computers with no games (I dont have any ZX spectrum roms).

**Comment by eduardofilo** [comment]
It is SVM, not SVN the extension of the blank file. Try again with that.

**Comment by lagwagone** [comment]
@eduardofilo thanks - well spotted on the typo. They're showing up as expected now :)
